### PR TITLE
Update crowdin workflow config

### PR DESCRIPTION
**Pull request by rickycodes**
this adds `metamaskbot` as value for the environment variable `GITHUB_ACTOR`

the other changes will mean the commits go to `metamaskbot`

we could also sign the commits so that they are `verified` later if we want:

https://github.com/crowdin/github-action/blob/master/README.md?plain=1#L121-L123

but I don't think it's required and i don't want to add another `secret` for this.

### CODE REVIEW

**Review by adonesky1 — DISMISSED** [review]
LGTM

**Review by adonesky1 — DISMISSED** [review]

**Review by jpuri — DISMISSED** [review]

File: .github/workflows/crowdin_action.yml [hunk 1]
```diff
@@ -25,6 +25,9 @@ jobs:
       with:
         upload_translations: true
         download_translations: true
+        token: ${{ secrets.METAMASKBOT_CROWDIN_TOKEN }}
```
[the remark this answers is not in the document]
Comment (Gudahtt) [reply .github/workflows/crowdin_action.yml:28]: I can see in the crowdin action that it gets passed to the CLI as the option `--token`, but that option isn't documented for the CLI :shrug:

[the remark this answers is not in the document]
Comment (rickycodes) [reply .github/workflows/crowdin_action.yml:28]: I just noticed this token being used in other configs on gh. It's an input for the action:

https://github.com/crowdin/github-action/blob/master/action.yml#L152-L154

It sets an option that the `crowdin` cli uses here: https://github.com/crowdin/github-action/blob/master/entrypoint.sh#L239-L241

`GITHUB_TOKEN` is used for auth _with_ github (we know that this is working for pull requests), but in order for the `crowdin` cli to also use PAT when _it_ pushes it needs _this_ `token`

If you check the `crowdin` cli help dialog you can see it's a flag:

```
crowdin push --help
...
CONFIG OPTIONS:
  -T, --token=...          Personal access token required for authentication
``` 

It does beg the question: "why do you need both?". I think the idea is you could use one PAT for opening PRs and another for `crowdin` cli pushes so you can have separate users doing each.

[the remark this answers is not in the document]
Comment (Gudahtt) [reply .github/workflows/crowdin_action.yml:28]: I see, so this is to ensure the `metamaskbot` is used for the commits as well. Makes sense.

Is this just a "nice to have", or is this related to problem where the CLA action fails to run?

[the remark this answers is not in the document]
Comment (rickycodes) [reply .github/workflows/crowdin_action.yml:28]: My hope is that it's _both_. That the token thing fixes the push issues we're seeing with the failed CLA action runs and the other config changes will give the commits to `metamaskbot`. That said I am bit iffy on the latter so I am going to reach out to crowdin to confirm.

[the remark this answers is not in the document]
Comment (Gudahtt) [reply .github/workflows/crowdin_action.yml:28]: Confirmation would be good! I can approve now anyway though, so we can test this out in the meantime.

**Review by Gudahtt — DISMISSED** [review]

File: .github/workflows/crowdin_action.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rickycodes) [reply .github/workflows/crowdin_action.yml:28]: they've actually confirmed that the token used here is _not_ a PAT for github, but that it's functionality equivalent to : https://github.com/MetaMask/metamask-extension/blob/develop/crowdin.yml#L2. I've asked how to get the pushes properly attributed. I'm going to add a do not merge label here until I hear back.

**Review by adonesky1 — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**